plugins/tech-radar/src/api.ts

```diff
@@ -34,11 +34,17 @@ export interface RadarQuadrant {
 export interface RadarEntry {
   key: string; // react key
   id: string;
-  moved: MovedState;
-  quadrant: RadarQuadrant;
-  ring: RadarRing;
+  quadrant: string;
   title: string;
   url: string;
+  history: Array<RadarEntryHistory>;
+}
+
+export interface RadarEntryHistory {
+  date: Date;
+  ring: string;
```

Comment:
Thanks you for this normalization, previous types where a bit weird 😅 

Think we should call it `ringId` instead?

Reply:
renamed to `ringId`